composer.json

```
@@ -1,61 +1,61 @@
 {
-    "name": "whilesmart/laravel-user-devices",
-    "description": "Package for managing user devices across our projects",
-    "license": "MIT",
-    "authors": [
-        {
-            "name": "WhileSmart",
-            "email": "engineering@whilesmart.com"
-        }
-    ],
-    "autoload": {
-        "psr-4": {
-            "Whilesmart\\LaravelUserDevices\\": "src"
-        }
-    },
-    "extra": {
-        "laravel": {
-            "providers": [
-                "LaravelUserDevicesServiceProvider"
-            ]
-        }
-    },
-    "require-dev": {
-        "orchestra/testbench": "^10.4",
-        "zircote/swagger-php": "^5.1",
-        "nunomaduro/collision": "^8.8",
-        "laravel/pint": "^1.22",
-        "fakerphp/faker": "^1.24"
-    },
-    "require": {
-        "laravel/sanctum": "^4.1"
-    },
-    "autoload-dev": {
-        "psr-4": {
-            "Workbench\\App\\": "workbench/app/"
-        }
-    },
-    "scripts": {
-        "pint": [
-            "./vendor/bin/pint"
-        ],
-        "pint:test": [
-            "./vendor/bin/pint --test"
-        ],
-        "post-autoload-dump": [
-            "@clear",
-            "@prepare"
-        ],
-        "clear": "@php vendor/bin/testbench package:purge-skeleton --ansi",
-        "prepare": "@php vendor/bin/testbench package:discover --ansi",
-        "build": "@php vendor/bin/testbench workbench:build --ansi",
-        "serve": [
-            "Composer\\Config::disableProcessTimeout",
-            "@build",
-            "@php vendor/bin/testbench serve --ansi"
-        ],
-        "lint": [
-            "@php vendor/bin/pint --ansi"
-        ]
+  "name": "whilesmart/laravel-user-devices",
+  "description": "Package for managing user devices across our projects",
+  "license": "MIT",
+  "authors": [
+    {
+      "name": "WhileSmart",
+      "email": "engineering@whilesmart.com"
+    }
+  ],
+  "autoload": {
+    "psr-4": {
+      "Whilesmart\\LaravelUserDevices\\": "src"
+    }
+  },
+  "extra": {
+    "laravel": {
+      "providers": [
+        "Whilesmart\\LaravelUserDevices\\LaravelUserDevicesServiceProvider"
```

```suggestion
        "Whilesmart\\UserDevices\\UserDevicesServiceProvider"
```


for the package name it's useful but in the code itself I figured out mentioning laravel everywhere is not of any help, we should not use this pattern across  the board.